internal/commands/inspect_image.go

```diff
@@ -91,6 +96,13 @@ func logDetails(remote *pack.ImageInfo, local *pack.ImageInfo, cfg config.Config
 	}
 }
 
+type process struct {
+	PType   string
```

Comment:
Why not just `Type` here? `lifecycle.Process` already does it.

Reply:
type is a reserved word in golang

Reply:
I had assumed that was the reason, but I saw that `lifecycle.Process` has a `Type` field.